### PR TITLE
S1 support

### PR DESCRIPTION
**Pull request by brikendr**

### CODE REVIEW

**Review by eskilgh — COMMENTED** [review]
Is it worth considering refactoring the Ace implementation to a IpBaseDevice class and extending Ace and See from that one?

**Comment by brikendr** [comment]
> Is it worth considering refactoring the Ace implementation to a IpBaseDevice class and extending Ace and See from that one?

Good point @eskilgh 👍🏼

**Review by pkhansen — APPROVED** [review]
Good stuff! Agree with Eskil on the IpBaseDevice

File: src/components/device/ace.ts [hunk 1]
```diff
@@ -1,1160 +1,31 @@
...
-  transport: IGrpcTransport;
-  /** @ignore */
-  locksmith: Locksmith;
+export default class Ace extends IpBaseDevice {
   /**
    * Comercial product name for this controller class.
    *
    * @type {string}
-   * @memberof Ace
+   * @memberof See
```
Comment (eskilgh) [inline src/components/device/ace.ts:17]: Should be Ace no?

File: src/components/device/ace.ts [hunk 2]
```diff
@@ -1,1160 +1,31 @@
...
 
   /**
-   * Creates an instance of Ace.
+   * Creates an instance of ACE/L1.
    * @param {*} wsdDevice The wsdd device instance retrieved during discovery.
    * @param {IGrpcTransport} transport Grpc transport instance.
-   * @param {EventEmitter} cameraDiscoveryEmitter Emitter instance sending attach & detach events for Huddly ACE cameras.
-   * @memberof Ace
+   * @param {EventEmitter} cameraDiscoveryEmitter Emitter instance sending attach & detach events for Huddly L1 cameras.
+   * @memberof See
```
Comment (eskilgh) [inline src/components/device/ace.ts:26]: see above

**Review by eskilgh — CHANGES_REQUESTED** [review]

File: src/components/device/ace.ts (hunk 1 above)
Comment (brikendr) [reply src/components/device/ace.ts:17]: Yes, good catch.

File: src/components/device/ace.ts (hunk 2 above)
Comment (brikendr) [reply src/components/device/ace.ts]: @eskilgh done

**Review by eskilgh — APPROVED** [review]
:tada:

**Pull request merged**